Mathlib/Algebra/Order/Group/Defs.lean

```diff
@@ -62,23 +63,14 @@ instance (priority := 100) OrderedCommGroup.toOrderedCancelCommMonoid [OrderedCo
 example (α : Type u) [OrderedAddCommGroup α] : CovariantClass α α (swap (· + ·)) (· < ·) :=
   IsRightCancelAdd.covariant_swap_add_lt_of_covariant_swap_add_le α
 
--- Porting note: this instance is not used,
--- and causes timeouts after lean4#2210.
--- It was introduced in https://github.com/leanprover-community/mathlib/pull/17564
--- but without the motivation clearly explained.
```

Comment:
That's my mathlib3 PR. Considering the slow TC synthesis, it really shouldn't be an instance.